servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/DefaultHostPriorityStrategy.java

```diff
@@ -61,7 +62,8 @@ private <T extends PrioritizedHost> List<T> rebuildWithPriorities(final List<T>
         TreeMap<Integer, Group> groups = new TreeMap<>();
         for (T host : hosts) {
             if (host.priority() < 0) {
-                LOGGER.warn("Found illegal priority: {}. Dropping priority grouping data.", host.priority());
+                LOGGER.warn("{}: Found illegal priority: {}. Dropping priority grouping data.",
```

Comment:
we typically specify the unmet criteria in the log message too (ie. `priority >= 0`)